### ISSUE TITLE
Change PVH Device Warning Behavior

### ISSUE THREAD

**Issue by zellchristensen** [QubesOS/qubes-issues#6108]
**The problem you're addressing (if any)**
When attaching a Device in the Devices tab of the Qube Settings, when the user clicks OK to save the settings, an error popup will appear if the Qube is set to PVH mode saying Devices cannot be attached in PVH mode, then the Settings window closes.

**Describe the solution you'd like**
Add the message to the top of the Device window (Similar to the message about devices and memory balancing) and grey out/make inaccessible the Available Devices section (such as when a Qube is running) when a Qube is set to PVH mode.

**Where is the value to a user, and who might that user be?**
Increases usability and clarity for the user.

**Describe alternatives you've considered**
Don't close the Settings window after the user clicks OK on the error popup so the user can correct the issue themselves by removing the devices.

OR

Have the error popup occur as soon as a device is added instead of waiting until the user clicks apply/ok.

OR

Have the error popup occur as soon as the Device tab is selected/opened (similar to the popup when opening Firewall Rules with networking disabled)

**Additional context**
Many people are desensitized to popup messages especially when only a single choice is given leading them to just click the choice without reading the prompt.
Currently, if a user fails to read the popup message, they may assume the settings are saved, then attempt to start the qube with the assumption that the devices they added were added when in fact they were not due to it being in PVH mode.
The user would have to repeat their steps and be mindful enough to re-read the popup, then open the settings again when the information could instead be more easily available to the user.

**Comment by andrewdavidwong** [comment]
Another idea is to say something like:

> You assigned a device to this qube, which requires changing the virtualization mode to HVM. Would you like to change the virtualization mode to HVM? [Yes] [No]

If the user selects Yes, change it to HVM.

If the user selects No, execute the current behavior.

**Comment by marmarta** [comment]
But the disable-device-tab is how it works right now in Qubes 4.0? ( see commit https://github.com/QubesOS/qubes-manager/commit/ebc7426cabd2e9f2dd4848d9767d4035a812bb59 ) 
I tested a VM with PVH enabled before, and one in which I set virt mode to PVH, and both disabled devices and added a warning. I also can't set mode to PVH after adding devices. 

What is the exact scenario, with setting, steps etc.?
